### DISCUSSION TITLE
How to work with loading library via CDN?

### DISCUSSION THREAD

**Discussion by codebeaulieu** [ueberdosis/tiptap#1000] [General Questions]
I'm adding a WYSIWYG editor to an existing application, we've been using CDN and bundling for our entire application thus far and every library we've used has worked fine, but I cannot figure out how to use this library using CDN.

In our scripts area (along with our other working scripts), I add the reference to the js

<script src="https://unpkg.com/tiptap@1.32.1/dist/tiptap.min.js" type="text/javascript"></script>

Then I attempt to follow the [basic-setup](https://github.com/ueberdosis/tiptap#basic-setup) but the `new Editor` reference is undefined. When that didn't work I tried "using" the Editor by adding `Vue.use(Editor);` but that causes a failure as well.

**Is it possible to use this library using the CDN? If so, what am I missing?**

**Comment by hanspagel** [comment]
Oh, yes! That doesn’t work well with v1.

Good news though, it’s working perfectly fine with the upcoming v2: https://blog.ueber.io/post/tiptap-2-0-beta/

And congratulations on opening issue #1000 😅

**Reply by codebeaulieu** [reply]
> And congratulations on opening issue #1000 😅

hilarious, when I first saw that I thought I was the first to ask a question and the numbering just started at 1k lol...

> Good news though, it’s working perfectly fine with the upcoming v2

1. hmm, alrighty, from what I gathered the beta is only open to sponsors, correct?
2.  If so, is it in a fairly usable state and what's the sponsorship obligation to be added?
3. Last question, is there a rough ETA on the release of V2?

I'm currently installing a much more primitive WYSIWYG editor, but your repo checks all of the boxes that I'd like to use. :)

**Reply by hanspagel** [reply]
There’s only one index for issues, PRs and questions on GitHub. So actually, it’s the 1000th something. 😅

1. Yes, we’re giving sponsors early-access.
2. It’s definitely in a usable state. There are 101 pages of documentation, 65 interactive examples, ~300 tests … It’s way more robust than v1 ever was. 😅  There are few things missing, the biggest of them are menus/toolbars, but depending on your use case it’s fairly easy to build them and there are already people porting over v1 menus to v2.
3. We started with v2 in August and aimed to release it in January, so there’s already a delay. We’re not good with estimates and we still can't work full-time because of this pandemic thingy. 😬 Currently, we have to major things we’d like to build in private: better React support + the interfaces/menus/toolbars/whatever you call them.
